### DISCUSSION TITLE
733

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#912] [Comments]
# 733

빅테크 회사분들의 시니어분들과 사석에서 만날때와 스타트업으로 이직하신 시니어분들을 만날때 서로 대화의 주제가 다름을 체감한다. 예를 들면, 우리팀의 주니어가 평균 레이턴시 1초인 API를 0.1초로 개선했다 를 이야기한다고 하면 여전히 빅테크에 계신 분들은 그 주니어가 기술적으로 뛰어난 사람임을 이야기하고, 어떻게 개선했는지가 주력이라고 한다면, 스타트업으로 이직하신 시니어분들은 "1초인데 왜 개선하지…?" 라는 이야기를 한다는 것이다. 두 그룹의 시니어분들 다 몇년 전까지는 비슷한 규모의 회사를 다녔음에도 말이다. 하루 수백만 ~ 수천만건의 호출이 발생하는 API이기 때문에 이를 0.1초로 개선한 것은 서비스 전체적인 개선이니 당연히 그 시기의 개발팀에 있어서는 정말 중요한 작업이다. 그리고 2년전까지..

https://jojoldu.tistory.com/733

**Comment by ldss3sang** [comment]
좋은 글 감사합니다! 잘 읽었습니다!

**Comment by hwanseok-dev** [comment]
잘 읽었습니다 감사합니다

**Comment by SaeWooKKang** [comment]
스타트업에서 어떤 경험을 쌓아야 하는지 좋은 인사이트 얻고 갑니다. 감사합니다.

**Comment by 2Ho0** [comment]
좋은 글 잘 읽었습니다!!!

**Comment by gukin-han** [comment]
저 같이 공부를 하고 취업을 준비하는 사람들도 고민해보면 좋을만한 내용입니다! 👍

**Comment by mystyle2006** [comment]
공감이 많이 되는 글이군요..

**Comment by jws1837** [comment]
너무 공감했습니다. 주니어로서 이러한 생각 하에 스타트업에서만 경험할 수 있는 것에 집중하자는 생각을 하면서도  내심 불안했는데, 이렇게 한 번 짚어주시니 너무 감사드립니다.

**Comment by choewy** [comment]
정말 많은 공감을 받았고, 인사이트를 얻게 된 것 같습니다. 항상 좋은 글 감사합니다!

**Comment by chrisjhkim** [comment]
스타트업은 아니지만 최근 두개의 신규 프로젝트를 진행했었습니다.
꽤나 타이트한 일정과 인력으로, 단기 목표에 따라 속도와 퀄리티 우선순위를 다르게 할 필요가 있었습니다.
첫번째 프로젝트에서는 몸소 시행착오를 겪으며 고생을 하였는데 
두번째 프로젝트에서는 나름 완급 조절을 하며 중국어 공부가 필요할땐 중국어를, 영어 공부가 필요할땐 영어공부를 했던것 같네요

이 글을 그 전에 읽었으면 조금 더 도움이 되지 않았을까 싶네요 

좋은글 감사합니다.

**Comment by andrewkimdev** [comment]
맞아요 맞아요, 자전거 탈 때에도 1단 기어 써야 할 때와 5단 기어 써야 할 때가 다르죠. 좋은 글 감사합니다.
